backend/os/file.go

```diff
@@ -329,7 +329,7 @@ func (f *File) copyWithName(name string, location vfs.Location) (vfs.File, error
 		return nil, err
 	}
 
-	if err := utils.TouchCopy(newFile, f); err != nil {
+	if err := utils.TouchCopyBuffered(newFile, f, 0); err != nil {
```

Comment:
See the Jira Ticket and my most recent comment to John on this. Seems like Os does not have filesystem options so for now I just have it passing in 0 to use the utils.TouchCopyMinBufferSize. I will update this code to use that constant directly so that it is less confusing by passing 0. Overall some opportunity to refactor and standardize how we set buffer size outlined in the ticket comment.